docs/source/1.0/spec/aws/aws-query-protocol.rst

```diff
@@ -58,6 +58,81 @@ See
             }
         }
 
+.. _aws.protocols#awsQueryError-trait:
+
+-------------------------------------
+``aws.protocols#awsQueryError`` trait
+-------------------------------------
+
+Summary
+    Provides the value in the "Code" distinguishing field and HTTP response
```

Comment:
Distinguishing field is a little vague. Maybe something like: Provides a custom "Code" value for awsQuery errors and an HTTP response code. The "Code" of an `awsQuery` error is used by clients to determine which type of error was encountered.

Reply:
It might be worth linking to the section that describes how the default Code and status code is computed.